lib/plugins/features/movies/controller.js

```
@@ -6,3 +6,16 @@ exports.create = (payload) => {
   return new Movie().save(payload)
   .then((movie) => new Movie({ id: movie.id }).fetch());
 };
+
+exports.findAll = (query) => {
+  if (query.min_year && query.max_year) {
+    return Movie.where('release_year', '>=', query.min_year)
+    .where('release_year', '<=', query.max_year).fetchAll();
+  }
+
+  if (query.title) {
```

handling for year filtering and title filtering looks good! but it seems to be limited to either being able to do year or title, not both. and years is also a bit limited because you have to pass in both min and max (and the validator doesnt indicate this requirement so it would just fail silently to the user).

what do you think about making this a bit more flexible to be able to handle any permutation? such as supplying `min_year` and `title` or maybe just `max_year`?

---

good call, will update 👍